src/options.jl

```diff
@@ -0,0 +1,8 @@
+Base.@kwdef mutable struct PresolveOptions{T}
```

Comment:
It's probably worth calibrating these tolerances with what other solvers use at some point. For example, Gurobi uses 1e-6 for primal/dual infeasibility and 1e-5 for integrality*.

*At least these are the global default tolerances, they could be different in certain spots like presolve.

Reply:
I'm relatively agnostic w.r.t default values, as long as they are clearly documented (that I haven't done yet).
The reason for using `eps(T)` is that it's a reasonable choice regardless of the arithmetic.

Reply:
I'm also agnostic at this point. Making them small seems like a good idea now, while we're building out new routines.

Reply:
Presolve tolerances should be tight.

For instance, consider
```
10 x + 10 y = 0
 1e-6 <= x <=  2e-6
-2e-6 <= y <= -1e-6
```
with absolute tolerances `1e-6`. The problem is obviously feasible with `x = -y = 1e-6`.

However, presolve may fix `x` to `1e-6` and `y` to `-2e-6`, which gives `10x + 10y = -1e-5`, and the problem would be declared infeasible.

Obviously there's always a risk that this would happen, but tighter tolerances reduce this by a lot.

Reply:
I'm fine with keeping tolerances tight. This example is also showing the utility of rescaling the problem early in presolve.